apps/test-netzer-encoder/src/main.cpp

```diff
@@ -0,0 +1,7 @@
+#include "NetzerEncoder.h"
+#include "mbed.h"
+
```

Comment:
also make sure test app reads both angle 
 and angular velocity

Reply:
Netzer only gives the absolute position information, would I be interpolating the angular velocity from this data @younesr1 ?